jenkins/janitor.py

```diff
@@ -109,18 +109,20 @@ def clear_resources(project, col, resource):
         if resource.condition:
             base.append('--%s=%s' % (resource.condition, col))
 
-        print "Try to kill %s - %s" % (col, list(items))
-        if subprocess.call(base + list(items)) != 0:
+        print 'Try to kill %s - %s' % (col, list(items))
+        try:
+            subprocess.check_call(base + list(items))
+        except subprocess.CalledProcessError as e:
             err = 1
-            print "Error try to delete %s - %s" % (col, list(items))
+            print >>sys.stderr, 'Error try to delete resources: %s' % e.output
```

Comment:
emm, done but not tested locally yet.. since my own project does not raise the exception and I cannot wipe a pr project locally with my account. Gonna throw it in and check the next run.